linkml_runtime/utils/schemaview.py

```diff
@@ -243,20 +241,10 @@ def __hash__(self) -> int:
             self._hash = hash(self.__key())
         return self._hash
 
-    @lru_cache(None)
-    def namespaces(self) -> Namespaces:
```

Comment:
moved this down to where similar functions that pull out all ____ (classes, slots, enums, etc.) are